Makefile

```diff
@@ -126,6 +126,11 @@ compose-up:
 	docker-compose up -d openpitrix-db && sleep 20 && docker-compose up -d
 	@echo "compose-up done"
 
+.PHONY: compose-up-app
+compose-up-app:
+	docker-compose up -d openpitrix-db && sleep 20 && docker-compose -f docker-compose-app.yml up -d
```

Comment:
`docker-compose -f docker-compose-app.yml up -d openpitrix-db `

Reply:
I means this is better:
`docker-compose -f docker-compose-app.yml up -d openpitrix-db && sleep 20 && docker-compose -f docker-compose-app.yml up -d`